tests/cases/clear_with_depth.amber

```diff
@@ -0,0 +1,51 @@
+#!amber
+# Copyright 2019 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHADER vertex vtex_shader PASSTHROUGH
+SHADER fragment frag_shader GLSL
+#version 430
+
+layout(location = 0) in vec4 color_in;
+layout(location = 0) out vec4 color_out;
+layout(location = 1) in vec4 color_in_1;
+layout(location = 1) out vec4 color_out_1;
+
+void main() {
+  color_out = color_in;
+}
+END
+
+BUFFER depthStencil_buf FORMAT D32_SFLOAT_S8_UINT
+BUFFER img_buf FORMAT B8G8R8A8_UNORM
```

Comment:
I'm tempted to say this should have dimensions, but right now the code assumes all colour attachments use the pipeline's framebuffer width and height.  That will be addressed in an issue I'm filing.

Reply:
The colour and depth buffers get their size from the pipeline framebuffer size.